lib/pages/home/home_page.dart

```diff
@@ -33,7 +34,7 @@ class HomePage extends StatelessWidget {
       ],
       body: ListView(
         children: [
-          if (authProvider.isAuthenticated) ProfileCard(),
+          if (authProvider.isAuthenticated) const ProfileCard(),
```

Comment:
I believe that adding `const`, in this case, will result in an undesired behavior.
As I experienced it, if a anonymous user is authenticated than the `ProfileCard` won't show up.